CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## `v1.0.0` 2022-10-28
```

Comment:
```suggestion
## `v0.1.0` 2022-10-28
```

Shouldn't a `v1.0.0` by convention incl at least a minimal feature set? Would reserve the `v1.0.0` for our MVP.